director/director_api.go

```diff
@@ -155,29 +150,23 @@ func CreateDirectorScrapeToken() (string, error) {
 	// We assume this function is only called on a director server,
```

Comment:
Same here, if the function only gets referred once, let's remove the function and put logic to where it was being used.